service/api/prettyprint.go

```diff
@@ -377,21 +386,25 @@ func PrettyExamineMemory(address uintptr, memArea []byte, format byte) string {
 		colFormat = "%08b"
 	case 'o':
 		cols = 8
-		colFormat = "%04o" // Always keep one leading zero for octal.
+		colFormat = "0%03o" // Always keep one leading zero for octal.
```

Comment:
The number in the format string should be `colBytes*number_of_character_in_a_byte_for_the_current _format`

Reply:
done

Reply:
Where?

Reply:
Oh, I revert this. 
---------------
I suggest use 1byte zero padding to format the number, it will be shorter
and looks a little better, for example, a 8-byte hexidecimal, 0x12345678
may be better than 0x0000000012345678.

Reply:
But then columns aren't going to be aligned.

Reply:
Yes ... done